src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -212,9 +212,6 @@ public static void Main()
                 // (10,13): error CS8117: Invalid operand for pattern match; value required, but found '<null>'.
                 //         if (null is dynamic t) { } // null not allowed
```

Comment:
Looks like this text is out of date.

Reply:
Fixed in a new iteration.

---
In reply to: [180253333](https://github.com/dotnet/roslyn/pull/26035#discussion_r180253333) [](ancestors = 180253333)